ui/00_message/examine_message/common/common_examine.xml

```diff
@@ -348,40 +357,55 @@ Unlikely to be opened.</translation>
   </message>
   <message>
     <original>敵はあわてて逃げ出した！</original>
+    <translation>The enemy panicked and ran away!</translation>
   </message>
   <message>
     <original>胞子が集まり始めている――</original>
+    <translation>Spores are starting to gather</translation>
   </message>
   <message>
     <original>汚染植物が出現――！</original>
+    <translation>Infected plants appear!</translation>
```

Comment:
Just "appear" sounds strange", "are appearing" or "have appeared" might be better?